backend/api/routes/boosterpacks/logic.go

```diff
@@ -119,8 +120,17 @@ func OpenBoosterPack(userID, tournamentID string, boosterPackData domain.Booster
 	var err error
 	if boosterPackData.BoosterType == domain.BoosterTypeDraft {
 		// Vanilla booster
-		cards, err = GenerateVanillaBoosterPack(boosterPackData)
+		exists, err := boostergen.CheckIfBoosterExists(boosterPackData.SetCode)
 		if err != nil {
+			log.Debug().Err(err).Send()
+		}
+		if exists {
+			cards, err = boostergen.GenerateBoosterFromJson(boosterPackData.SetCode)
+		} else {
+			cards, err = GenerateVanillaBoosterPack(boosterPackData)
+		}
```

Comment:
Should just throw an error if it doesn't exist. Let's remove this function since we got the new, better implementation working.